pkg/operator/utils/topology.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/legacy-cloud-providers/vsphere"
+)
+
+func GetDatacenters(config *vsphere.VSphereConfig) ([]string, error) {
```

Comment:
Don't you need the changes in pkg/operator/vspherecontroller/checks/check_nodes.go too?
https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/123/files#diff-872043e7b350e043c36bbae56b122ac24f16df367350b832376f0e3a2a23582eL239-R240

Reply:
Good catch! Will fix.

Reply:
fixed.